plugin-server/src/kafka/admin.ts

```diff
@@ -0,0 +1,30 @@
+import { AdminClient, CODES, GlobalConfig, IAdminClient, LibrdKafkaError } from 'node-rdkafka'
+
+import { status } from '../utils/status'
+
+export const ensureTopicExists = async (adminClient: IAdminClient, topic: string) => {
+    // Ensures that a topic exists. If it doesn't, it will be created. If it
+    // does, this is a no-op. We use -1 for the number of partitions and
+    // replication factor as this will use the default values configured in
+    // the Kafka borker config.
```

Comment:
```suggestion
    // the Kafka broker config.
```

I'm the Kafka borker though 😆